extensions/projection/README.md

```diff
@@ -94,3 +121,38 @@ This object represents the centroid of the Item Geometry.
 | ------------------- | ------ | ------------------------------------------------------------ |
 | lat                 | number | The latitude of the centroid.  |
 | lon                 | number | The longitude of the centroid. |
+
+## Best Practices
+
+There are clearly a number of projection extension fields, with potentially overlapping functionality. This section attempts to 
+give an overview of which ones you should consider using. They fit into three general categories:
+
+- **Description of the coordinate reference system:** [EPSG code](#projepsg) is the default, but it is just a reference to known
+projection information. [WKT2](#projwkt2) and [PROJJSON](#projprojjson) are two options to fully describe the projection information.
```

Comment:
+ "For example, the MODIS Sinusoidal projection does not have an EPSG code, but can be described using WKT2 or PROJJSON"